docs/changelog.md

```diff
@@ -4,8 +4,16 @@ title: Changelog
 permalink: /changelog/
 ---
 
-# 101.0.0 (In Development)
-* [Commits](https://github.com/mozilla-mobile/android-components/compare/v100.0.0...main)
+# 102.0.0 (In Development)
+* [Commits](https://github.com/mozilla-mobile/android-components/compare/v101.0.0...main)
+* [Milestone](https://github.com/mozilla-mobile/android-components/milestone/149?closed=1)
+* [Dependencies](https://github.com/mozilla-mobile/android-components/blob/main/buildSrc/src/main/java/Dependencies.kt)
+* [Gecko](https://github.com/mozilla-mobile/android-components/blob/main/buildSrc/src/main/java/Gecko.kt)
+* [Configuration](https://github.com/mozilla-mobile/android-components/blob/main/.config.yml)
+
+
+# 101.0.0
+* [Commits](https://github.com/mozilla-mobile/android-components/compare/v100.0.0...v101.0.0)
 * [Milestone](https://github.com/mozilla-mobile/android-components/milestone/148?closed=1)
 * [Dependencies](https://github.com/mozilla-mobile/android-components/blob/main/buildSrc/src/main/java/Dependencies.kt)
 * [Gecko](https://github.com/mozilla-mobile/android-components/blob/main/buildSrc/src/main/java/Gecko.kt)
```

Comment:
These links need to be updated from `main` to `v101.0.0`

Reply:
Thanks for catching this